src/Merchants/Content/Merchant/Api/MerchantProductController.php

```diff
@@ -0,0 +1,314 @@
+<?php declare(strict_types=1);
```

Comment:
@shyim Should I move this class somewhere else? Not too sure about the destination

Reply:
Should be okay :)